cla_backend/apps/checker/admin.py

```diff
@@ -6,6 +6,7 @@
 from django.http.response import HttpResponseRedirect
 from checker.models import ReasonForContacting, CallbackTimeSlot
 from checker.forms import CallbackTimeSlotCSVUploadForm
+import datetime as dt
```

Comment:
Why is datetime being aliased as dt here?
I think using the name of the package when referring to the package is easier to read unless you are importing another datetime package in which aliasing will make sense as there will be a name collision.